docs/src/templates/pages/styleguide/ui-templates/menu/menu-header-variations.hbs

```diff
@@ -0,0 +1,33 @@
+---
+title: Menu Header Variations
+section-title: UI Templates
+docs: true
+
+navgroup: styleguide
+navsub: ui-templates
+navactive: template-menu
+---
+
+<p><em>The <code>badge</code> component is an optional mixin within Fozzie — if you'd like to use it in your project you can include it by adding <code>@include badge();</code> into your SCSS dependencies file.</em></p>
+
+<p>Badges can be used to display small pieces of information.</p>
+
+<h2 class="sg-sectionHeading">Delivery Open</h2>
+{{#>demo-block }}
+{{> je-menu-header status='je-menu-status-delivery-open' star-rating='30' star-rating-count=566 }}
```

Comment:
Any reason the one star rating param has quotes and the other doesn't?

Reply:
I've made them more consistent now. Zero star ratings are still passed in as a string as the param is used to build up a CSS class which expects `'00'`, passing that as an int removes one of the zeroes 😢